src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/CallGCSuppressFinalizeCorrectlyTests.cs

```diff
@@ -471,6 +471,27 @@ End Sub
             VerifyBasic(code);
         }
 
+        [Fact]
+        public void Disposable_ImplementedExplicitly_NoDiagnostic()
+        {
+            var code = @"
+using System;
+
+public class ImplementsDisposableExplicitly : IDisposable
+{
+    void IDisposable.Dispose()
+    {
+        Dispose(true);
+        GC.SuppressFinalize(this);
+    }
+
+    protected virtual void Dispose(bool disposing)
+    {
+    }
+}";
+            VerifyCSharp(code);
```

Comment:
VB test?

Reply:
Totally blanked on that, especially since VB can name their disposal implementation arbitrarily.